### PR TITLE
Router/lucas

### PR DESCRIPTION
**Pull request by lulup52**
we clean the router and add a homme button to "make yout cocktail"

### CODE REVIEW

**Comment by lulup52** [comment]
we try to merge all feature together

File: src/App.jsx [hunk 1]
```diff
@@ -1,161 +1,28 @@
...
-          </div>
-        </div>
-      </div>
+      <Router>
+        <Switch>
+          <Route path="/cocktails/create">
+            <MakeYourCocktail />
+          </Route>
+          <Route path="/cocktails">
+          </Route>
```
Comment (rocambille) [inline src/App.jsx]: this is empty

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (lulup52) [reply src/App.jsx]: im working on it

File: src/App.jsx [hunk 2]
```diff
@@ -1,161 +1,29 @@
...
-          </div>
-        </div>
-      </div>
+      <Router>
+        <Switch>
+          <Route path="/cocktails/create">
+            <MakeYourCocktail />
+          </Route>
+          <Route path="/cocktails">
+          <ChooseYourCocktail />
```
Comment (rocambille) [inline src/App.jsx]: fix indentation ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve :)

**Comment by rocambille** [comment]
looks like your last merge broke the indentation fix

**Pull request merged**